javaServices/coreJavaServices/src/main/java/joliex/meta/MetaJolie.java

```diff
@@ -403,8 +403,8 @@ private Value getOutputPort( OutputPortInfo portInfo ) {
 		} else {
 			response.getFirstChild( "location" ).setValue( "undefined" );
 		}
-		if( portInfo.protocolId() != null ) {
-			response.getFirstChild( "protocol" ).setValue( portInfo.protocolId() );
+		if( portInfo.protocol() != null ) {
+			response.getFirstChild( "protocol" ).setValue( portInfo.protocol().toString() );
```

Comment:
I wouldn't invoke OLSyntaxNode.toString(), this is misleading, since I'd expect an InlineTreeExpression to print out the entire tree, not just the root.

Consider keeping the `protocolId` method to do this maybe?